src/psyclone/tests/domain/lfric/lfric_config_test.py

```diff
@@ -64,6 +64,19 @@
 COMPUTE_ANNEXED_DOFS = false
 supported_fortran_datatypes = real, integer, logical
 default_kind = real: r_def, integer: i_def, logical: l_def
+precision_map = i_def: 4,
```

Comment:
This reminds me that the User Guide (https://psyclone.readthedocs.io/en/stable/configuration.html#options) needs to be updated to document these new settings.